src/OrchardCore/OrchardCore.Contents.TagHelpers/ContentLinkTagHelper.cs

```diff
@@ -15,6 +15,7 @@ namespace OrchardCore.Contents.TagHelpers
     [HtmlTargetElement("a", Attributes = ContentLinkEdit)]
     [HtmlTargetElement("a", Attributes = ContentLinkRemove)]
     [HtmlTargetElement("a", Attributes = ContentLinkCreate)]
+    [HtmlTargetElement("a")]
```

Comment:
No need of ContentLinkTagHelper to Target `<a>`

Use asp.net tag helper instead